icons/nextra.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nextra</title><path d="m22.226.29-.989.988c-5.089 5.09-13.342 5.09-18.43 0L1.816.29A1.05 1.05 0 0 0 .335 1.774l.988.988c5.09 5.09 5.09 13.343 0 18.432l-.988.988a1.05 1.05 0 1 0 1.484 1.484l.988-.988c5.09-5.089 13.342-5.089 18.431 0l.989.988a1.05 1.05 0 0 0 1.484-1.484l-.988-.988c-5.09-5.09-5.09-13.343 0-18.432l.988-.988A1.05 1.05 0 0 0 22.226.29Z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nextra</title><path d="M23.677 22.111A1.109 1.109 0 0 1 22.894 24a1.1 1.1 0 0 1-.782-.324l-.983-.984A12.823 12.823 0 0 0 12 18.912a12.82 12.82 0 0 0-9.128 3.78l-.983.984a1.13 1.13 0 0 1-1.565 0 1.108 1.108 0 0 1 0-1.565l.983-.983c5.034-5.034 5.034-13.224 0-18.257l-.983-.983A1.099 1.099 0 0 1 0 1.106 1.107 1.107 0 0 1 1.888.324l.984.982c5.034 5.035 13.224 5.034 18.257 0l.983-.983a1.108 1.108 0 0 1 1.565 0 1.108 1.108 0 0 1 0 1.565l-.984.983c-5.033 5.033-5.033 13.223 0 18.257l.984.983z"/></svg>
```

This should fix the issues.

Reply:
![Untitled](https://github.com/simple-icons/simple-icons/assets/35000807/a150178d-0d9f-4b0c-9178-388fd13b81e4)
With fixed logo^